docs/changelog/3_x.rst

```diff
@@ -91,6 +91,12 @@ commands to use our testing channel and the beta release.
 
 This process will involve some downtime, specifically during steps 2 and 3.
 
+**Pre-1.0 Instances**
+
+If you're still running pre-1.0 EdgeDB instances like 1.0-beta3, and would like
+to upgrade to 3.0, you would need a dump-restore towards version 2.x first,
+followed by another dump-restore to 3.0.
```

Comment:
```suggestion
If you're still running pre-1.0 EdgeDB instances (e.g., 1.0-beta3) and want
to upgrade to 3.0, you would need a dump/restore to version 2.x first,
followed by another dump/restore to 3.0.
```

Also, would it be more accurate to say you need to first **upgrade** to 2.x with a dump/restore and then **upgrade** to 3.0 with a dump/restore? I'm afraid leaving out the word "upgrade" might imply you can just dump/restore without an upgrade... unless that is your intent here.

Reply:
Right, it should be "upgrade" ❤️